Functions.Templates/Templates/HttpTrigger-Python/README.md

```diff
@@ -0,0 +1,54 @@
+# Python HttpTrigger
+
+The Azure Functions Python HTTP Trigger creates an endpoint: `<function-host>/api/<function-name>`. When an HTTP request is made against that endpoint, the `main()` function in `__init__.py` is called.
+
+> Throughout this documentation variables in angle brackets `<>` are placeholders and will be different in your application.
+
+## Default Structure
```

Comment:
Reminder to self- rename to Template Structure

Reply:
I think "Default" is fine.